Implementation/service/FeedbackAnalyzer.py

```diff
@@ -0,0 +1,91 @@
+from data.Database import connect_to_db
+from core.SentimentAnalyzer import analyze_sentiment
+from notification.notification import add_notification
+from datetime import datetime
+
+class FeedbackAnalyzer:
```

Comment:
Why is the class SentimentAnalyzer in core directory but Feedback Analyzer in service directory? How are these categorized?